code/game/objects/structures/girders.dm

```diff
@@ -374,3 +374,51 @@
 	girder_state = GIRDER_BUILDING1_WELDED
 	reinforcement = GIRDER_REINF_PLASTEEL
 	max_integrity = 500
+
+//Not a child of /girder/ because it would need a lot of stuff overwritten or changed on the parent
+/obj/structure/shuttle_girder
+	name = "girder"
```

Comment:
to clarify are all walls meltable or no

Reply:
Everything on a shuttle (except the floors and the girders) can be destroyed. The walls melt, but the girders left behind cannot be destroyed. They just become un-dense and with a broken sprite.